Tests/Unit/Domain/Repository/CategoryRepositoryTest.php

```diff
@@ -48,8 +48,13 @@ public function setUp()
             ->shouldBeCalled()
             ->willReturn($query);
 
-        $psrContainer = new FailsafeContainer();
-        $objectManager = new ObjectManager($psrContainer, new Container($psrContainer));
+        if (class_exists(FailsafeContainer::class)) {
```

Comment:
Wenn in den UnitTests irgendwas nicht läuft, wegen der fehlenden Dependencies, dann bau den Test bitte um als FunctionalTests. Dort ist nach parent::setUp() alles vorhanden, was Du brauchst.